python/tests/test_storage.py

```
@@ -0,0 +1,226 @@
+  assert first.checksum.algorithm == 'md5'
+  assert first.checksum.value == 'deadbeef'
+  assert second.checksum.algorithm == 'md5'
+  assert checksum_paths == [
+    '//rucio/file?cks.type=adler32',
+    '//rucio/file?cks.type=md5',
+    '//rucio/file?cks.type=md5',
+  ]
+
+
+def test_owned_environment_auth_is_closed_with_client():
+  auth = AuthContext.bearer(token='secret')
```

Ruff (0.16.2)

[error] Possible hardcoded password assigned to argument: "token"

(S106)